src/main/java/com/semmle/jira/addon/config/init/PluginEnabledHandler.java

```diff
@@ -41,15 +45,25 @@ public void destroy() {
   }
 
   @EventListener
-  public void onPluginEnabled(PluginEnabledEvent event) {
+  public void onPluginEnabled(PluginEnabledEvent event) throws FactoryException, IOException {
     Plugin plugin = event.getPlugin();
     if (LGTM_PLUGIN_KEY.equals(plugin.getKey())) {
-      try {
-        issueTypeManager.createIssueType(
-            ISSUE_TYPE_NAME, "Issue type for managing LGTM alerts", 0l);
-      } catch (IllegalStateException e) {
-        // Issue Type already created
-      }
+      createLgtmWorkflow();
     }
   }
+
+  private static void createLgtmWorkflow() throws IOException, FactoryException {
+    ClassLoader classLoader = PluginEnabledHandler.class.getClassLoader();
+    InputStream is = classLoader.getResourceAsStream("workflow/workflow.xml");
+    String workflowXml = IOUtils.toString(is);
```

Comment:
This method seems deprecated. I guess because it does not specify the character encoding. https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/IOUtils.html#toString(java.io.InputStream)

Best use `IOUtils.toString(is, StandardCharsets.UTF_8)`   
(repeated 3 times)